crates/rust-analyzer/tests/slow-tests/tidy.rs

```diff
@@ -229,6 +229,7 @@ Apache-2.0 OR BSL-1.0
 Apache-2.0 OR MIT
 Apache-2.0 WITH LLVM-exception OR Apache-2.0 OR MIT
 Apache-2.0/MIT
+BlueOak-1.0.0 OR MIT OR Apache-2.0
```

Comment:
Not sure where this is coming from but given its still dual licensed under MIT and apache I don't think it matters